operation/manifest_test.go

```diff
@@ -5,8 +5,67 @@ import (
 
 	"github.com/stretchr/testify/require"
 	"gopkg.in/yaml.v2"
+	yamlv3 "gopkg.in/yaml.v3"
 )
 
+const fullSpringMusicYaml = `applications:
+- name: spring-music
+  buildpacks:
+  - java_buildpack_offline
+  env:
+    SPRING_CLOUD_PROFILE: dev
+  routes:
+  - route: spring-music-egregious-porcupine-oa.apps.example.org
+  services:
+  - name: my-sql
+  stack: cflinuxfs3
+  command: java
+  disk_quota: 1G
+  health-check-type: http
+  health-check-http-endpoint: /health
+  instances: 2
+  log-rate-limit-per-second: 100MB
+  memory: 1G
+  timeout: 60
+`
+
+const minimalSpringMusicYaml = `applications:
+- name: spring-music
+  buildpacks:
+  - java_buildpack_offline
+  no-route: true
+  stack: cflinuxfs3
+  health-check-type: port
+  health-check-http-endpoint: /
+  instances: 1
+  memory: 1G
+`
+
+const fullSpringMusicYamlV2 = `applications:
+  - name: spring-music
+    buildpacks:
+      - java_buildpack_offline
+    env:
+      SPRING_CLOUD_PROFILE: dev
+    routes:
+      - route: spring-music-egregious-porcupine-oa.apps.example.org
+    services:
+      - name: my-sql
+        binding_name: mysql
+        parameters:
+          name: mysql
+      - oauth2
```

Comment:
I have already included the mentioned cases as well.

Reply:
I missed that oauth2 service, thanks for adding that!